client/components/account-balances/test/index.test.tsx

```diff
@@ -290,3 +292,64 @@ describe( 'AccountBalancesTabPanel', () => {
 		expect( jpyPendingAmount ).toHaveTextContent( '¥20' );
 	} );
 } );
+
+describe( 'BalanceTooltip', () => {
+	test( 'renders the correct tooltip text for the available balance', () => {
+		const expectedTooltipText = fundTooltipStrings.available;
+
+		render(
+			<BalanceTooltip
+				label="Available funds tooltip"
+				content={ expectedTooltipText }
+			/>
+		);
+
+		const tooltipButton = screen.getByRole( 'button', {
+			name: 'Available funds tooltip',
+		} );
+		fireEvent.click( tooltipButton );
+
+		screen.getByText( expectedTooltipText );
+	} );
+
+	test( 'renders the correct tooltip text for a negative available balance', () => {
+		const expectedTooltipText = fundTooltipStrings.availableNegativeBalance;
+
+		render(
+			<BalanceTooltip
+				label="Available funds tooltip"
+				content={ expectedTooltipText }
+			/>
+		);
+
+		const tooltipButton = screen.getByRole( 'button', {
+			name: 'Available funds tooltip',
+		} );
+		fireEvent.click( tooltipButton );
+
+		screen.getByText( expectedTooltipText );
+	} );
+
+	test( 'renders the correct tooltip text for the pending balance', () => {
+		const delayDays = 17;
+		// Insert the delayDays value into the expected tooltip text.
+		const expectedTooltipText = fundTooltipStrings.pending.replace(
```

Comment:
Should we use hardcoded string in the unit test? This is so the unit test 'locks' the code and can detect if there is changes in the code.

With using variable `fundTooltipStrings.pending` here, when it's changed in the future, this unit test will not fail and probably won't aware where are the places that are affected by the change.

What do you think?

Reply:
> This is so the unit test 'locks' the code and can detect if there is changes in the code.

As I understand it, that's the pattern – the test should hard-code the expectations and ideally only depend on the component being tested.

Reply:
Thanks @shendy, @haszari! I've since refactored the tests in 845e8738d4b9179d87c8886b1c8e6df1336b6789 with this in mind, since the previous tests made no sense now that the strings have been moved out of `strings.ts`.

I've kept the documentation URLs as imported variables from `strings.ts` however, as I think these should be kept in a separate file for maintainability purposes.